frontend-public/src/components/LandingPage.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import { Divider, Row, Col, Button } from "antd";
 
+import QuestionSidebar from "./common/QuestionsSidebar";
```

Comment:
"Home" might be a better place to put this rather than common. I doubt other screens will use this in the future.

Reply:
I can move it but its also used on both of the userDashboard pages.

Reply:
Ok. We can move it later if we start adding a lot of other functionality.